taxonomies/ingredients.txt

```diff
@@ -20561,7 +20569,7 @@ en:natural flavouring, natural flavour, natural flavor, natural flavoring, natur
 ca:Aroma natural, aromes naturals
 da:naturlig aroma
 de:natürliches Aroma
-es:aroma natural, aromas naturales
+es:aroma natural, aromas naturales, aromas naturales conservadores
```

Comment:
"aromas naturales conservadores" is probably not a valid entry, more of an OCR issue where it's missing a separator between "aromas naturales" and "conservadores"

Reply:
I'll remove for the moment until I find which product it came from